### PR TITLE
tests: improve tests self documentation for nested test suite

### PR DESCRIPTION
**Pull request by sergiocazzolato**

### CODE REVIEW

**Comment by mvo5** [comment]
Fails because of typo:
```
tests/nested/classic/hotplug/task.yaml:7:31: "teh" is a misspelling of "the"
```

**Comment by bboozzoo** [comment]
This one is weird, but I don't think it's related. Basically EBADMSG when trying to mount?
```
2021-03-08T19:10:14.4535498Z + uc20-create-partitions --mount ./gadget-dir '' /dev/loop5
2021-03-08T19:10:14.4536899Z panic: cannot mount filesystem "/dev/loop5p4" at "/run/mnt/ubuntu-save": bad message
2021-03-08T19:10:14.4537479Z 
2021-03-08T19:10:14.4537865Z goroutine 1 [running]:
2021-03-08T19:10:14.4538287Z main.main()
2021-03-08T19:10:14.4539326Z 	/home/gopath/src/github.com/snapcore/snapd/tests/lib/uc20-create-partitions/main.go:80 +0x574
```

**Review by bboozzoo — APPROVED** [review]

File: tests/nested/core20/tpm/task.yaml [hunk 1]
```diff
@@ -40,51 +39,49 @@ execute: |
...
-    # 2. good recovery systems is populated with current systems
+    echo "2. good recovery systems is populated with current systems"
     nested_exec "sudo sed -i -e 's/good_recovery_systems=.*/good_recovery_systems=/' /var/lib/snapd/modeenv"
 
     boot_id="$( nested_get_boot_id )"
     nested_exec "sudo reboot" || true
     nested_wait_for_reboot "${boot_id}"
-    # the test will race with snapd updating the modeenv, wait for the modenv to
-    # have a good state
+    echo "Check the test will race with snapd updating the modeenv, wait for the modenv to have a good state"
```
Comment (mvo5) [inline tests/nested/core20/tpm/task.yaml:79]: This reads a bit strange, I would avoid using "Check" as the first word here.

**Review by mvo5 — APPROVED** [review]
Thank you, one small suggestion about wording

**Pull request merged**